File: RunNET/Testa.cs
// This is the minimal skeleton of a program that
// can be run using RunNET: a Program class
// with a Main method

using System;

public class Program
{
  // This method should be declared as: 
  //
  // Main(string[] arguments)
  //
  // or
  //
  // Main()
  //
  // The return type can be void or int
  public static int Main(string[] arguments)
  {
    Console.WriteLine("Hello world from C#!");
    for (int index = 0; index < arguments.Length; index++)
    {
      Console.WriteLine("Argument({0}) = '{1}'", index, arguments[index]);
    }
    return arguments.Length;
  }
}

File: RunNET/Testa.vb
' This is the minimal skeleton of a program that
' can be run using RunNET: a Program class
' with a Main method

Imports System

Public Class Program
  ' This method should be declared as: 
  '
  ' Main(arguments() As String)
  '
  ' or
  '
  ' Main()
  '
  ' It can be either a Sub or a function returning Integer
  Public Shared Function Main(ByVal arguments() As String) As Integer
    Console.WriteLine("Hello world from VB.NET!")
    For index As Integer = 0 To arguments.Le - 1
      Console.WriteLine("Argument({0}) = '{1}'", index, arguments(index))
    Next
    Return arguments.Length
  End Function
End Class
